app/design/adminhtml/default/default/template/rakuten/rakutenpay/info/boleto.phtml

```diff
@@ -20,7 +20,7 @@
 
 <script>
 <?php
-    $http = new \RakutenPay\Resources\Http();
+    $http = new \RakutenPay\Resources\RakutenPay\Http();
```

Comment:
Namespace "RakutenPay"\Resources\"RakutenPay"\Http

Is duplicated: The first name no longer exists because the module will be refactored. Unification of modules.

Reply:
👍